module4/labs/Week4/SecurityConcerns.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: Security Concerns Lab
+---
+
+<section class='call-to-action' markdown='1'>
+
+Encourage students to reach out to each other during this lesson.  If they complete it, they will be almost there on implementing authorization; we don't need them to get through the whole thing, but it would be good to wrap up with a short discussion of how we would compare two digests (database value to user input).
+
+</section>
+
+## Implementing Security Measures
+
+In the lesson today, we discussed concepts.  Now, we are going to put these concepts into practice! Open your MVC Message Logger application (the mod3 project).  Add the following features:
+
+### Remove Connection Strings
+
+Using either secrets management or environment variables, remove the database connection string from your application.
+
```

Comment:
I would put a step in here to double check that you are still able to see all of your messages after removing the connection string! I could totally see students removing it, but not checking that the new solution works 😆

Reply:
hahaha totally 😆